google-map-marker.html

```diff
@@ -366,7 +366,7 @@
         subtree: true
       });
 
-      var content = this.innerHTML.trim();
+      var content = this.children[0];
```

Comment:
My original question of whether this will work under native shadow dom is still bothering me. `this.children[0]`will give you the marker's light dom, but you've added the `<div>` wrapper to the element's shadow dom. This will require users to use a single wrapper for their marker content, so I'm not sure what the internal wrapper does.

```
<google-map-marker>
  <div>...</div>
</google-map-marker>
```

One thing you could do is use (`this.$.contentwrapper`), and the id: `<template><content  id="contentwrapper"></content></template>`, and pull the distributed nodes: `this.$.content.getDistributedNodes()` to see what users have put inside the marker element. Then, wrap all that in a single `<div>`?